bin/metanorma

```diff
@@ -176,15 +161,15 @@ if Gem.win_platform?
   # END of HACK
 
   require 'ffi-libarchive-binary' # to preload libarchive-13.dll
-end
 
-Net::HTTP.class_eval do
-  alias _use_ssl= use_ssl=
+  Net::HTTP.class_eval do
```

Comment:
@alexeymorozov can we remove this SSL hack?

Reply:
We can after we find a solution for ocra. It's now applied only for Windows.

Reply:
Ah I see, thanks!